### ISSUE TITLE
New submissions for Wednesday, 4 September 2024 (showing 67 of 67 entries )

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#19]
## An NV- center in magnesium oxide as a spin qubit for hybrid quantum technologies
- **Keywords:** ['hybrid quantum', 'hybrid quantum']
 - **Authors:** Vrindaa Somjit, Joel Davidsson, Yu Jin, Giulia Galli
 - **Subjects:** Subjects:
Materials Science (cond-mat.mtrl-sci); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2409.00246
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2409.00246
 - **Abstract**
 Recent predictions suggest that oxides, such as MgO and CaO, could serve as hosts of spin defects with long coherence times and thus be promising materials for quantum applications. However, in most cases specific defects have not yet been identified. Here, by using a high-throughput first-principles framework and advanced electronic structure methods, we identify an NV-like center in MgO with favorable electronic and optical properties for hybrid quantum technologies. We show that this defect has stable triplet ground and excited states, with singlet shelving states enabling optical initialization and spin-dependent readout. We predict several properties, including absorption, emission, and zero-phonon line energies, as well as zero-field splitting tensor, and hyperfine interaction parameters, which can aid in the experimental identification of this defect. Our calculations show that due to a strong pseudo-Jahn Teller effect and low frequency phonon modes, the NV- center in MgO is subject to a substantial vibronic coupling. We discuss design strategies to reduce such coupling and increase the Debye-Waller factor, including the effect of strain and the localization of the defect states. We propose that the favorable properties of the NV- defect, along with the technological maturity of MgO, could enable hybrid classical-quantum applications, such as spintronic quantum sensors and single qubit gates.
## Noise-induced phase transitions in hybrid quantum circuits
- **Keywords:** ['hybrid quantum', 'hybrid quantum']
 - **Authors:** Shuo Liu, Ming-Rui Li, Shi-Xin Zhang, Shao-Kai Jian, Hong Yao
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Disordered Systems and Neural Networks (cond-mat.dis-nn); Statistical Mechanics (cond-mat.stat-mech); Strongly Correlated Electrons (cond-mat.str-el)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2401.16631
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2401.16631
 - **Abstract**
 The presence of quantum noises inherent to real physical systems can strongly impact the physics in hybrid quantum circuits with local random unitaries and mid-circuit measurements. The quantum noises with a size-independent occurring probability can lead to the disappearance of a measurement-induced entanglement phase transition and the emergence of a single area-law phase. In this work, we investigate the effects of quantum noises with size-dependent probabilities $q=p/L^{\alpha}$ where $\alpha$ represents the scaling exponent. We have identified a noise-induced entanglement phase transition from a volume law to a power (area) law in the presence (absence) of measurements as $p$ increases when $\alpha=1$. With the help of an effective statistical model, we reveal that the phase transition is of first-order arising from the competition between two types of spin configurations and shares the same analytical understanding as the noise-induced coding transition. This unified picture further deepens the understanding of the connection between entanglement behavior and the capacity of information protection. When $\alpha \neq 1$, one spin configuration always dominates regardless of $p$ and thus the phase transition disappears. Moreover, we highlight the difference between the effects of size-dependent bulk noise and boundary noises. We validate our analytical predictions with extensive numerical results from stabilizer circuit simulations.
## Transfer Learning Based Hybrid Quantum Neural Network Model for Surface Anomaly Detection
- **Keywords:** ['hybrid quantum']
 - **Authors:** Sounak Bhowmik, Himanshu Thapliyal
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2409.00228
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2409.00228
 - **Abstract**
 The rapid increase in the volume of data increased the size and complexity of the deep learning models. These models are now more resource-intensive and time-consuming for training than ever. This paper presents a quantum transfer learning (QTL) based approach to significantly reduce the number of parameters of the classical models without compromising their performance, sometimes even improving it. Reducing the number of parameters reduces overfitting problems and training time and increases the models' flexibility and speed of response. For illustration, we have selected a surface anomaly detection problem to show that we can replace the resource-intensive and less flexible anomaly detection system (ADS) with a quantum transfer learning-based hybrid model to address the frequent emergence of new anomalies better. We showed that we could reduce the total number of trainable parameters up to 90% of the initial model without any drop in performance.
## A hybrid quantum-classical fusion neural network to improve protein-ligand binding affinity predictions for drug discovery
- **Keywords:** ['hybrid quantum']
 - **Authors:** L. Domingo, M. Chehimi, S. Banerjee, S. He Yuxun, S. Konakanchi, L. Ogunfowora, S. Roy, S. Selvaras, M. Djukic, C. Johnson
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2309.03919
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2309.03919
 - **Abstract**
 The field of drug discovery hinges on the accurate prediction of binding affinity between prospective drug molecules and target proteins, especially when such proteins directly influence disease progression. However, estimating binding affinity demands significant financial and computational resources. While state-of-the-art methodologies employ classical machine learning (ML) techniques, emerging hybrid quantum machine learning (QML) models have shown promise for enhanced performance, owing to their inherent parallelism and capacity to manage exponential increases in data dimensionality. Despite these advances, existing models encounter issues related to convergence stability and prediction accuracy. This paper introduces a novel hybrid quantum-classical deep learning model tailored for binding affinity prediction in drug discovery. Specifically, the proposed model synergistically integrates 3D and spatial graph convolutional neural networks within an optimized quantum architecture. Simulation results demonstrate a 6% improvement in prediction accuracy relative to existing classical models, as well as a significantly more stable convergence performance compared to previous classical approaches.
